Automattic/simplenote-macos
# Interlink References: Mark II

### Details:
In this PR we're implementing **Click** support over reference rows, in the Metrics UI.

@eshurakov Last one sir! Thank you!!

Closes #663 

### Test
1. Log into your account!
2. Right click over **Note A** in your notes list: click over Copy Internal Link
3. Click over **Note B** and paste the contents of your clipboard
4. Click back over **Note A**
5. Click over the top right ( i ) icon

- [x] Verify that moving the mouse over the reference cells switches the mouse cursor (should be a pointing finger!)
- [x] Verify that clicking over **Note B** updates both the Selected Note in the Notes List, and the Editor

### Release
`RELEASE-NOTES.txt` was updated in 28ea507 with:

> Note Details now also display inbound references #836
